src/eradiate/scenes/surface/_dem.py

```diff
@@ -543,4 +542,7 @@ def from_mesh(
                 bsdf=opacity_bsdf,
             )
 
+        else:
+            raise TypeError(f"Scene geometry is not recognized. {geometry}")
```

Comment:
Let's hint the user at what went wrong in a more explicit and concise manner:
```suggestion
            raise TypeError(f"geometry must be PlaneParallelGeometry or SphericalShellGeometry, got {type(geometry)}")
```